doc/operation-and-maintenance/Cluster-configuration-and-node-management.md

```diff
@@ -126,32 +106,37 @@ mongooseimctl leave_cluster
 
 It only makes sense to use it if the node is the part of a cluster, e.g `join_cluster` was called from that node before.
 
-The successful output from the above command starts with `You have successfully left the node from the cluster`.
+Similarly to `join_cluster` a warning and a question will be displayed unless option `-f` is added to the command.
```

Comment:
Similarly to `join_cluster` a warning and a question will be displayed unless the option `-f` is added to the command.